astroid/rebuilder.py

```diff
@@ -104,13 +104,17 @@ def __init__(
             self._parser_module = parser_module
         self._module = self._parser_module.module
 
-    def _get_doc(self, node: T_Doc) -> Tuple[T_Doc, Optional[str]]:
+    def _get_doc(self, node: T_Doc) -> Tuple[Optional[str], Optional[nodes.Const]]:
```

Comment:
If we deprecate `doc`, we should consider if we want to switch the return values.
I.e. `doc_node, doc` instead of `doc, doc_node`.

Reply:
As said, deprecation should go into another PR